sql-join/magic-mallrats.sql

```diff
@@ -0,0 +1,12 @@
+select "inventoryId",
+      "filmId",
+      "rentalId",
+      films.title as title,
```

Comment:
This alias isn't necessary since it uses the column name, which is already "title".